src/plugins/console/public/styles/_index.scss

```diff
@@ -7,3 +7,19 @@
 
 @import 'app';
 @import 'components/index';
+
+.ace_editor.ace_autocomplete {
+  resize: horizontal;
+
+  .ace_scroller {
+    right: 6px !important;
```

Comment:
This seems necessary to me. Without this, `.ace_scroller` element covers and hides the right edge of `.ace_editor.ace_autocomplete` element on Safari and Firefox. Firefox seems to allow users to click a resizer behind elements, but Safari does not. So, it seems at the moment that the default styling makes the resizer clickable on Chrome, Edge, and Firefox but not Safari.

Reply:
In my testing in the latest versions of Chrome, Firefox, Safari, Opera, and Edge, removing this style doesn't appear to cause any issues (in fact, it looks better as it removes the aforementioned text collisions). I'm still able to resize the panel in Safari. From what I can see, it's the styles on lines 22–25 below that ensure the panel is resizable in Safari. Assuming I'm not missing something, my suggestion would be to remove this.

Reply:
@MichaelMarcialis Thank you very much for the clarification. That sounds like my Safari 17.2.1 behaves differently from yours (17.4.x?). Would you mind if I ask you to check the following reproducer?

```html
data:text/html,<div%20style="position:fixed;width:280px;height:128px;resize:both;border:solid%201px;overflow:hidden;"><div%20style="position:absolute;right:0px;bottom:0px;">FOO<div></div>
```

I can resize the parent div element on my Chrome, Firefox, and Edge but can't on my Safari 17.2.1 due to the overlapping child div element like this.

https://github.com/elastic/kibana/assets/721858/3101b8c2-78b6-4587-8ff1-ed4d6035a3be

Because of this problem, I had to add this style. Without this style, the same problem prevents the resizer from being active on Safari 17.2.1 like this.

https://github.com/elastic/kibana/assets/721858/72147604-28d6-45fa-a57c-a8a05427501f

Reply:
@sakurai-youhei: Yes, I'm on Safari 17.4.1. Despite that, my Safari functions exactly the same as yours, and the inability to grab the resize handle can be reproduced using the example code you provided above. However, I do not see the same when testing locally after commenting out lines 14–20 in `_index.scss`. Instead, I see the following:

![CleanShot 2024-04-22 at 10 08 51](https://github.com/elastic/kibana/assets/3884767/ff07c11f-ce7a-47a0-a3b5-39ca91224e0d)

![CleanShot 2024-04-22 at 10 10 34](https://github.com/elastic/kibana/assets/3884767/b05f0845-1424-4080-a5de-9eaa303b0172)

The reason it still works is because there are default inline styles applied to `.ace_scroller`, as seen in the inspector below. When they aren't being overridden by us, everything works fine in Safari. I suggest we just roll with those defaults.

![CleanShot 2024-04-22 at 10 10 34](https://github.com/elastic/kibana/assets/3884767/728b0e6c-0d68-47f5-a027-80925053bdd0)

Reply:
@MichaelMarcialis Thank you for the detailed information.

Thanks to that, I find that the [Show scroll bars](https://support.apple.com/guide/mac-help/change-appearance-settings-mchlp1225/mac) setting makes a difference [HERE](https://github.com/thlorenz/brace/blob/v0.11.1/index.js#L1084-L1119). The `Automatically` leads 0 while the `Always` leads 15, and the calculated scroll bar width is propagated to the `right` inline style on `.ace_scroller`. What a beautiful trap! :)

So now I can start thinking of the rest. Once I get some idea, I will prepare and push a commit dealing with this point.

Reply:
Finally, I have found this style is unnecessary thanks to you. I appreciate your time on this very much!